src/dotnet-install.ps1

```diff
@@ -283,7 +283,7 @@ function Get-NormalizedChannel([string]$Channel) {
 
     switch ($Channel) {
         { $_ -eq "lts" } { return "LTS" }
-        { $_ -eq "current" } { return "current" }
+        { $_ -eq "sts" } { return "STS" }
```

Comment:
I don't think we should change the underlying channel name. What I understand from the earlier conversations around this was that this will be a purely cosmetic change. Otherwise, we would need to update aka.ms link generation and rename all the folders in the blob storage.

What we probably need is 
```suggestion
        { $_ -eq "sts" } { return "current" }
```

Reply:
agree